requirements/dev.txt

```diff
@@ -150,16 +150,6 @@ commonmark==0.9.1
     #   -c requirements/ci.txt
     #   -r requirements/ci.txt
     #   recommonmark
-coreapi==2.3.3
```

Comment:
Did you remove this manually? `coreapi` wasn't dropped in zgw-consumers 1.0.0 I think